### PR TITLE
Work with dpad and analogs

### PR DESCRIPTION
**Pull request by raelgc**
Browse in ES with all directionals, dpad and analogs. But it depends on https://github.com/RetroPie/RetroPie-Setup/pull/2529

### CODE REVIEW

**Comment by raelgc** [comment]
It depends on https://github.com/RetroPie/RetroPie-Setup/pull/2529

File: es-app/src/components/TextListComponent.h [hunk 1]
```diff
@@ -37,14 +37,14 @@ class TextListComponent : public IList<TextListData, T>
 	using IList<TextListData, T>::stopScrolling;
 
 	TextListComponent(Window* window);
-	
+
```
Comment (joolswills) [inline es-app/src/components/TextListComponent.h]: whitespace fixes / changes should be done in separate commit.

Comment (raelgc) [reply es-app/src/components/TextListComponent.h]: Oops, my bad! I'll remove them!

**Comment by joolswills** [comment]
Excluding my comments I have no issues with this but welcome comments. I am not as involved with ES Dev as others.

**Comment by raelgc** [comment]
@joolswills Just removed the whitespace/style fixes.

**Comment by joolswills** [comment]
Thanks. Will merge.

**Comment by hhromic** [comment]
This seems like a really handy contribution, how is the release cycle for EmulationStation in stable to bring the other side of the changes into standard RetroPie? :)

**Comment by metaColin** [comment]
> how is the release cycle for EmulationStation in stable to bring the other side of the changes into standard RetroPie?

I am also wondering how long before this feature gets into a release?
